jplag/src/main/java/de/jplag/reporting/Report.java

```diff
@@ -58,10 +61,18 @@ public void writeResult(JPlagResult result) throws ReportGenerationException {
         System.out.println("\nWriting report...");
         writeIndex();
         copyStaticFiles();
-        writeMatches(result.getComparisons(options.getMaximumNumberOfComparisons()));
+        writeMatches(result.getComparisons(getOptions().getMaximumNumberOfComparisons()));
         System.out.println("Report exported to " + reportDir.getAbsolutePath());
     }
 
+    private JPlagOptions getOptions() {
+        return options;
+    }
+
+    private Language getLanguage() {
+        return language;
+    }
+
```

Comment:
I think you should delete private getters. They are not needed.

Reply:
Yes, they are not needed. Their main usage here was during refactoring, I will inline them